### ISSUE TITLE
Esc key does not exit insert mode in notebooks opened from file browser or launcher

### ISSUE THREAD

**Issue by firai** [jupyterlab-contrib/jupyterlab-vim#94]
## Description

After entering insert mode from a notebook opened from the file browser or launcher, pressing <kbd>Esc</kbd> does not exit out of insert mode. Instead, it fires a "Cannot execute key binding 'Escape': command 'documentsearch:end' is not enabled." warning (see console log at bottom).

If the notebook is already opened when the JL page is refreshed, <kbd>Esc</kbd> exits out of insert mode as expected. In this case, pressing <kbd>Esc</kbd> in normal mode fires the warning above, without apparent user-facing side effects.

<kbd>Ctrl+]</kbd> is able to exit insert mode as a workaround.

I'm wondering if this has something to do with #90... @krassowski, do you know if this is expected to resolve itself when JL 4.1 is released?

## Context

- Jupyterlab-vim 4.0.0
- Jupyterlab 4.0.3
- Operating System and its version: Windows 11
- Browser and its version: Firefox 115 and Microsoft Edge 115

<details><summary>Console log</summary>
<pre>
Language pack 'English_United States' not valid! [jlab_core.0faecc546c73b7263d33.js:1:1441965](http://localhost:8888/static/lab/jlab_core.0faecc546c73b7263d33.js?v=0faecc546c73b7263d33)
Uncaught (in promise) Error: Editor not available
    modifyEditor http://localhost:8888/lab/extensions/@axlair/jupyterlab_vim/static/353.8b609c372dc07b2baac6.js?v=8b609c372dc07b2baac6:1
    updateLastActive http://localhost:8888/lab/extensions/@axlair/jupyterlab_vim/static/353.8b609c372dc07b2baac6.js?v=8b609c372dc07b2baac6:1
    x http://localhost:8888/lab/extensions/@axlair/jupyterlab_vim/static/353.8b609c372dc07b2baac6.js?v=8b609c372dc07b2baac6:1
    activate http://localhost:8888/lab/extensions/@axlair/jupyterlab_vim/static/353.8b609c372dc07b2baac6.js?v=8b609c372dc07b2baac6:1
    promise callback*activate http://localhost:8888/lab/extensions/@axlair/jupyterlab_vim/static/353.8b609c372dc07b2baac6.js?v=8b609c372dc07b2baac6:1
    promise http://localhost:8888/static/lab/jlab_core.0faecc546c73b7263d33.js?v=0faecc546c73b7263d33:1
    promise callback*activatePlugin http://localhost:8888/static/lab/jlab_core.0faecc546c73b7263d33.js?v=0faecc546c73b7263d33:1
    i http://localhost:8888/static/lab/jlab_core.0faecc546c73b7263d33.js?v=0faecc546c73b7263d33:1
    start http://localhost:8888/static/lab/jlab_core.0faecc546c73b7263d33.js?v=0faecc546c73b7263d33:1
    K http://localhost:8888/static/lab/812.b5f86ce00b402f3e2056.js?v=b5f86ce00b402f3e2056:1
    e http://localhost:8888/static/lab/main.2e96fbfd279ab7fd7ade.js?v=2e96fbfd279ab7fd7ade:1
    37559 http://localhost:8888/static/lab/main.2e96fbfd279ab7fd7ade.js?v=2e96fbfd279ab7fd7ade:1
    t http://localhost:8888/static/lab/main.2e96fbfd279ab7fd7ade.js?v=2e96fbfd279ab7fd7ade:1
    <anonymous> http://localhost:8888/static/lab/main.2e96fbfd279ab7fd7ade.js?v=2e96fbfd279ab7fd7ade:1
    <anonymous> http://localhost:8888/static/lab/main.2e96fbfd279ab7fd7ade.js?v=2e96fbfd279ab7fd7ade:1
[353.8b609c372dc07b2baac6.js:1:547](http://localhost:8888/lab/extensions/@axlair/jupyterlab_vim/static/353.8b609c372dc07b2baac6.js?v=8b609c372dc07b2baac6)
Cannot execute key binding 'Escape': command 'documentsearch:end' is not enabled. [jlab_core.0faecc546c73b7263d33.js:1:1608961](http://localhost:8888/static/lab/jlab_core.0faecc546c73b7263d33.js?v=0faecc546c73b7263d33)
</pre>
</details>

**Comment by firai** [comment]
> ```json
> {
>   "command": "vim:enter-normal-mode",
>   "keys": ["Escape"],
>   "selector": ".jp-Notebook"
> },
> ```
> 
> for notebook may do. Can you confirm if this helps?

Yes, this seems to work

**Comment by firai** [comment]
Upon further testing, it seems like command mode shortcuts, i.e., those starting with <kbd>Ctrl+O</kbd>, seem to also be borked in version 4.0.0, presumably after #90 was merged. I don't know if whether it's related or a separate issue that I should open a separate ticket for.

**Comment by krassowski** [comment]
Can you confirm that https://github.com/jupyterlab-contrib/jupyterlab-vim/pull/95 fixes both issues?

**Comment by firai** [comment]
Yes. Thanks!

**Comment by ianhi** [comment]
Should be fixed in the latest release. and hopefully this release will also work on conda-forge as well

**Comment by firai** [comment]
Hi @ianhi, I think this can be unpinned now. What do you think?
